packages/analytics-js/__tests__/components/configManager/ConfigManager.test.ts

```diff
@@ -123,19 +136,17 @@ describe('ConfigManager', () => {
       }),
     );
 
-    configManagerInstance.init();
+    configManagerInstance.getConfig();
 
     setTimeout(() => {
```

Comment:
A fixed timeout is not a proper method to ensure we are ready to verify the results. Instead, add a new property on the config manager, say `loaded`. Frequently poll that boolean and use the `done` callback to indicate test completion.

Reply:
That is not correct. We use global state with reactive abilities.

To check for a value change you can use the 
effect(() => { you expects here .... ; done()})

There is no need to poll.

Reply:
Using a signal would be a better solution.

Reply:
Done

Reply:
Done